src/main/java/core/basesyntax/strategy/PurchaseHandler.java

```diff
@@ -0,0 +1,25 @@
+package core.basesyntax.strategy;
+
+import core.basesyntax.db.FruitTransaction;
+import java.util.List;
+
+public class PurchaseHandler implements OperationHandler {
+    private static final String PURCHASE = "p";
+
+    @Override
+    public List<FruitTransaction> getFruitFinalQuantity(List<FruitTransaction> listOfAll,
+                                                        List<FruitTransaction> listOfFruits) {
```

Comment:
The method name 'getFruitFinalQuantity' does not clearly reflect the operation it performs, which is processing a purchase. Consider renaming it to better reflect its purpose, such as 'processPurchases'.